### ISSUE TITLE
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js)

### ISSUE THREAD

**Issue by harobed** [APIs-guru/graphql-voyager#123]
I have this error when I execute `npm run bundle`:

```
...
ERROR in ./src/components/settings/RootSelector.css
Module build failed (from ./node_modules/css-loader/dist/cjs.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
ValidationError: CSS Loader Invalid Options

options should NOT have additional properties

    at validateOptions (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/schema-utils/src/validateOptions.js:32:11)
    at Object.loader (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/css-loader/dist/index.js:44:28)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/webpack/lib/NormalModule.js:301:20
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at runSyncOrAsync (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:143:3)
    at iterateNormalLoaders (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:232:2)
    at iterateNormalLoaders (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at context.callback (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/postcss-loader/src/index.js:197:9
 @ ./src/components/settings/RootSelector.tsx 18:0-28
 @ ./src/components/settings/Settings.tsx
 @ ./src/components/Voyager.tsx
 @ ./src/components/index.tsx
 @ ./src/index.tsx
 @ multi ./src/polyfills.ts ./src/index.tsx

ERROR in ./src/components/viewport.css
Module build failed (from ./node_modules/css-loader/dist/cjs.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
ValidationError: CSS Loader Invalid Options

options should NOT have additional properties

    at validateOptions (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/schema-utils/src/validateOptions.js:32:11)
    at Object.loader (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/css-loader/dist/index.js:44:28)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/webpack/lib/NormalModule.js:301:20
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at runSyncOrAsync (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:143:3)
    at iterateNormalLoaders (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:232:2)
    at iterateNormalLoaders (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at context.callback (/Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/stephane/Work/github.com/APIs-guru/graphql-voyager/node_modules/postcss-loader/src/index.js:197:9
 @ ./src/components/Voyager.tsx 37:0-24
 @ ./src/components/index.tsx
 @ ./src/index.tsx
 @ multi ./src/polyfills.ts ./src/index.tsx
...
```

**Comment by harobed** [comment]
[`minimize: true`](https://github.com/APIs-guru/graphql-voyager/blob/master/build/webpack.js#L83) is [invalid option in `css-loader` (`v2.0.1`)](https://github.com/webpack-contrib/css-loader/tree/v2.0.1#options)

**Comment by IvanGoncharov** [comment]
Fixed in #125

**Comment by Musayeb** [comment]
k
